### PR TITLE
POL-650 Add orgId to PoliciesAction

### PR DESCRIPTION
**Pull request by gwenneg**
Same as [policies-engine#377](https://github.com/RedHatInsights/policies-engine/pull/377) with conflicts solved.

⚠️ Self-review in progress, I need to make sure I didn't screw up during the merge 😄

### CODE REVIEW

File: src/main/java/com/redhat/cloud/policies/engine/process/PoliciesAction.java [hunk 1]
```diff
@@ -21,12 +21,27 @@
...
+    private String orgId;
     private LocalDateTime timestamp;
     private Context context = new Context();
     private Set<Event> events = new HashSet<>();
 
+    private final boolean useOrgId;
+
+    public PoliciesAction() {
+        this(false);
+    }
```
Comment (gwenneg) [inline src/main/java/com/redhat/cloud/policies/engine/process/PoliciesAction.java:33]: This wasn't in the original PR.

**Comment by gwenneg** [comment]
@bennetelli I double-checked every change in this PR, it is equivalent to #377 (with one exception commented above). Is that OK with you if I push this branch onto the existing changes in #377?

**Comment by bennetelli** [comment]
> @bennetelli I double-checked every change in this PR, it is equivalent to #377 (with one exception commented above). Is that OK with you if I push this branch onto the existing changes in #377?

sure it is. You could even just do it and let me know afterwards :)

**Comment by gwenneg** [comment]
Pushed onto the other branch.

**Pull request closed without merging**